### PR TITLE
Refactor css

### PR DESCRIPTION
**Pull request by nathanSamosa**

### CODE REVIEW

File: src/client/styling/form.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
+/* FORM PAGE FORMATTING */
```
Comment (TheunisK) [inline src/client/styling/form.css:1]: Nice. I was going to suggest having separate files for styling so it stays manageable as the code expands.

File: src/server/controllers/pokemon.js [hunk 2]
```diff
@@ -30,6 +30,7 @@ const getPokeApi = async() => {
 }
 
 const createPokemons = async(req, res) => {
+    const {} = req.body
```
Comment (TheunisK) [inline src/server/controllers/pokemon.js:33]: Is this supposed to be empty?

Comment (nathanSamosa) [reply src/server/controllers/pokemon.js:33]: this was when Steve was talking about authorising the createPokemons fetch, req.body would contain an admin token or password. we dont fetch with a json body yet but when we do the authentication will be in there

Comment (TheunisK) [reply src/server/controllers/pokemon.js:33]: Ok yes makes sense.

File: src/client/App.css [hunk 3]
```diff
@@ -1,196 +1,13 @@
...
-  text-align: center;
-}
-
-.regInput {
-  font-weight: 100;
-}
-
-#registerLink {
-  grid-area: registerLink;
-}
```
Comment (Can-Tina) [inline src/client/App.css:192]: You've really refined it! Well done

File: src/server/controllers/pokemon.js (hunk 2 above)
Comment (glowkeeper) [reply src/server/controllers/pokemon.js:33]: Awesome :)

**Review by glowkeeper — APPROVED** [review]
Approved

**Pull request merged**